src/commands/variations/update.ts

```diff
@@ -65,11 +66,11 @@ export default class UpdateVariation extends UpdateCommand {
         } else {
             selectedVariation = await fetchVariationByKey(this.authToken, this.projectKey, featureKey, args.key)
         }
-
+        const feature = await fetchFeatureByKey(this.authToken, this.projectKey, featureKey) as Feature
         this.prompts.push(await getVariationVariablePrompt(
             this.authToken,
             this.projectKey,
-            featureKey,
+            feature._id,
```

Comment:
this seems like it would be an API bug, all the endpoints should accept either a key or an ID 🤔

Reply:
Even for query params? This field in the query params class called `isObjectIDorFalse` which makes it seem intentional

Reply:
Im just not sure why 😅

Reply:
Ideally all values could be accepted as keys, I'm assuming we made this choice to avoid querying the feature by key just to build the variable query params. We could consider updating the API, but that's out of scope of this work 😛